openpype/hosts/houdini/plugins/publish/validate_frame_range.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.hosts.houdini.api.action import SelectInvalidAction
+
+import hou
+
+
+class ValidateFrameRange(pyblish.api.InstancePlugin):
+    """Validate Frame Range.
+
+    Due to the usage of start and end handles,
+    then Frame Range must be >= (start handle + end handle)
+    which results that frameEnd be smaller than frameStart
+    """
+
+    order = pyblish.api.ValidatorOrder - 0.1
+    hosts = ["houdini"]
+    label = "Validate Frame Range"
+    actions = [SelectInvalidAction]
+
+    def process(self, instance):
+
+        invalid = self.get_invalid(instance)
+        if invalid:
+            nodes = [n.path() for n in invalid]
+            raise PublishValidationError(
+                "Invalid Frame Range on: {0}".format(nodes),
+                title="Invalid Frame Range"
+            )
+
+    @classmethod
+    def get_invalid(cls, instance):
+
+        if not instance.data.get("instance_node"):
+            return
+
+        rop_node = hou.node(instance.data["instance_node"])
+        if instance.data["frameStart"] > instance.data["frameEnd"]:
+            cls.log.error(
+                "Wrong frame range, please consider handle start and end.\n"
+                "frameEnd should at least be {}.\n"
```

Comment:
I think we can improve this error message.

Let's at least explain to the user what the issue is "Wrong frame range" - what is wrong? So the log could read:

> Invalid frame range because the instance start frame ({start}) is higher than the end frame ({end}).

It could also additionally log an `info` message when it's invalid saying:

> The ROP node render range is set to {frameStartHandle} - {frameEndHandle}
> The asset handles applied to the instance are start handle {handleStart} and end handle {handleEnd}

Just so that from the logs an artist could directly identify _what_ could be the cause of the issue.